src/ast/nodes/ExportDefaultDeclaration.js

```diff
@@ -28,60 +28,48 @@ export default class ExportDefaultDeclaration extends Node {
 		this.variable = this.scope.addExportDefaultDeclaration( this._declarationName || this.module.basename(), this );
 	}
 
-	// TODO this is total chaos, tidy it up
 	render ( code, es ) {
-		const treeshake = this.module.bundle.treeshake;
+		const remove = () => { code.remove( this.leadingCommentStart || this.start, this.next || this.end ); };
+		const removeExportDefault = () => { code.remove( this.start, declaration_start ); };
+		
+		const treeshakeable = this.module.bundle.treeshake && !this.included & !this.declaration.included;
```

Comment:
I guess it should be `&&` both times?

Reply:
Yes, I mistyped it. I will correct it later (3 hours)